editortask.h

```diff
@@ -47,6 +47,7 @@ class EditorTask : public Task
     unsigned int linesUntil(const char *end);
     void updateCursor();
     void changeSelection(unsigned int cursor_pos_new);
+	void runFile(const char *file);
```

Comment:
Whitespace error

Reply:
Weird how I don't see those errors in my text editor :/